CHANGES.txt

```diff
@@ -1,5 +1,6 @@
 5.0
  * Remove org.apache.cassandra.hadoop code (CASSANDRA-18323)
+ * Move cassandra related system properties and envs to an appropriate enum (CASSANDRA-17797)
```

Comment:
nit: always first entry :) 
 (and why we typically add it in post-review)

Reply:
will be fixed on commit